### DISCUSSION TITLE
Is it possible to get prebuilt Val compiler binaries?

### DISCUSSION THREAD

**Discussion by benedetto73** [hylo-lang/hylo#794] [Q&A] [answered]
Possibly Windows.
I would like to play with the compiler but I don't have too much time to build it.
(I briefly tried the VSCode dev container in Windows but didn't work).

Thanks.

**Comment by kyouko-taiga** [comment, marked as answer]
We don't have any stable version yet, so I'm not sure it would make a lot of sense. But as soon as we do we can surely imagine offering pre-built binaries.

**Reply by benedetto73** [reply]
Thanks.
Please consider that I am available to start working on open source projects in Val during my spare time if I had access to a compiler. Even a partially built one.

**Reply by srgx** [reply]
@benedetto73 You have time to work on project but no time to build compiler...

**Reply by kyouko-taiga** [reply]
@srgx It is a perfectly reasonable remark for someone willing to contribute to the standard library, test compiler features, implement additional libraries, develop tooling support, or do anything else not directly related to compiler code.

While we're not at a point where such contributions can be made using pre-built binaries, let's not discourage people willing to work on them once we do; they are also tremendously important.

**Reply by benedetto73** [reply]
@srgx I used to work in the MSVC++ compiler. But those days are long gone. I think I can contribute with the library and other tool work at this time.
